gateway/cert.go

```diff
@@ -470,10 +512,17 @@ func (gw *Gateway) getTLSConfigForClient(baseConfig *tls.Config, listenPort int)
 
 			if newConfig.ClientAuth >= tls.RequestClientCert {
 				newConfig.ClientCAs = x509.NewCertPool()
+				if dummyCA != nil {
+					// add a dummy CA as if CAs are not sent, browser popup appears prompting to select
+					// cert to authenticate. Peer will not have a cert signed by this dummy CA so popup
+					// will not appear. Peers can still connect with certs NOT signed by dummy CA so not
+					// breaking change
+					newConfig.ClientCAs.AddCert(dummyCA)
```

Comment:
That's interesting trick for sure. But I wonder how it differ from the case when ClientCAs are empty, and how it is related to popup 🤔

Reply:
I think that's the case from chrome documentation https://developer.chrome.com/docs/extensions/reference/certificateProvider/

`If the user aborts the authentication or no certificate matched the request, the TLS client authentication is aborted.` i.e. no popup and empty cert msg sent upstream.